### DISCUSSION TITLE
Dark Mode Toggle Switch

### DISCUSSION THREAD

**Discussion by VaibhavSharma24** [gatsbyjs/gatsby#38692] [Ideas / Feature Requests]
I want to add the Dark mode toggle switch on gatsby website I think it will make it more convenient for our users to scroll through in night and will enhance the overall user experience

**Comment by zhaohuanyuu** [comment]
you can use [themeui](https://theme-ui.com/) directly,or refer to [lekoArts‘s theme-utils](https://github.com/LekoArts/portfolio-v2/tree/main/src/themes-utils) and use it with your own css solution.

**Reply by VaibhavSharma24** [reply]
@zhaohuanyuu Okay thanks for approval one more thing I wanna know do I need to open issue first for this and then make a PR or directly I should make the PR ?

**Reply by zhaohuanyuu** [reply]
No, this is a business requirement and does not fall within the functional scope of the framework. You need to implement it yourself.
